contractsV5/kyberDAO/IKyberDAO.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity 0.5.11;
+
```

Comment:
this file should be in main contract folder.
not in Kyber DAO

Reply:
which contract calls claimReward from DAO? Only staker calls claimReward right?